render/gles2/renderer.c

```diff
@@ -52,6 +52,7 @@ static void gles2_begin(struct wlr_renderer *wlr_renderer, uint32_t width,
 
 static void gles2_end(struct wlr_renderer *wlr_renderer) {
 	gles2_get_renderer_in_context(wlr_renderer);
+	glFinish();
```

Comment:
Ugh. Is this really needed? This makes the compositor block until all drawing operations are done.

Reply:
My bad, this one wasn't supposed to go into this PR. It is required on nouveau, but I was planning to move it somewhere else, so that it only gets called when the cursor is changed.